pkg/vault-mon/prometheus.go

```diff
@@ -64,15 +64,15 @@ func PromWatchCerts(pkimon *PKIMon, interval time.Duration) {
 		Name: "x509_crl_length",
 		Help: "Length of certificate revocation list",
 	}, []string{"source"})
+	var promWatchCertsDuration = promauto.NewHistogram(prometheus.HistogramOpts{
```

Comment:
```suggestion
	promWatchCertsDuration := promauto.NewHistogram(prometheus.HistogramOpts{
```

Just looks funny to have everything else declared using `:=` but this uses `var` 😛

Reply:
agreed